libs/libc/stdio/lib_fclose.c

```diff
@@ -129,7 +129,7 @@ int fclose(FAR FILE *stream)
                                                (uintptr_t)stream);
           status = android_fdsan_close_with_tag(stream->fs_fd, tag);
```

Comment:
need move fdscan related code to stdio callback

Reply:
done